scripts/rs-enum.sh

```diff
@@ -16,14 +16,30 @@
 #
 # Alderlake:
 #$ ./rs-enum.sh 
-# Bus   Camera  Sensor  Node Type  Video Node    RS Link
-# ipu6  3       depth   Streaming  /dev/video52  /dev/video-rs-depth-3
-# ipu6  3       depth   Metadata   /dev/video53  /dev/video-rs-depth-md-3
-# ipu6  3       ir      Streaming  /dev/video56  /dev/video-rs-ir-3
-# ipu6  3       color   Streaming  /dev/video54  /dev/video-rs-color-3
-# ipu6  3       color   Metadata   /dev/video55  /dev/video-rs-color-md-3
-#
+#Bus	Camera	Sensor	Node Type	Video Node	RS Link
+# ipu6	0	depth	  Streaming	/dev/video4 	  /dev/video-rs-depth-0
```

Comment:
Can we change all IPU6 comments to be generic and also suitable to Jetson?

Reply:
this is generic, i can change "mipi" to "Tegra"

Reply:
OK Let's leave as is